doc/source/tutorials/Q-Factor.ipynb

```diff
@@ -0,0 +1,435 @@
+    "\n",
+    "* External Q: $$Q_e = \\omega_r \\left. \\frac{\\textrm{Energy Stored in the Resonant Circuit}}{\\textrm{Power Loss in the External Circuit}} \\right|_{\\omega=\\omega_r}$$\n",
+    "\n",
+    "* Loaded Q: $$Q_L = \\omega_r \\left. \\frac{\\textrm{Energy Stored in the Resonant Circuit}}{\\textrm{Total Power Loss}} \\right|_{\\omega=\\omega_r}$$\n",
+    "\n",
+    "The _loaded_ Q-factor $Q_L$, describes energy dissipation within the entire resonant system comprising of the resonator itself and the instrument used for observing resonances [3]. The term _loading_ refers to the effect that the external circuit has on measured quantities. Any loss mechanisms due to the external circuitry will have the effect of lowering the $Q$.\n",
+    "\n",
+    "The _unloaded_ Q-factor $Q_0$ is a characteristic of the resonator itself, in the absence of any loading effects caused by external circuitry (uncoupled). For most applications, the quantity that is desired is the unloaded Q-factor $Q_0$ which is determined by energy dissipation associated with the resonator only and therefore gives the best description of the resonant mode. Direct measurement of the unloaded Q of a resonator is generally not possible because of the loading effect of the measurement system. However, it is possible to estimate $Q_0$ from measurements of the frequency response of the loaded resonator.\n",
+    "\n",
+    "The energy dissipation in the external circuit is characterized by the _external_ Q-factor $Q_e$ and these three Q-factors are related by the relationship (deduced from the three expressions above):\n",
+    "\n",
+    "$$\n",
+    "\\frac{1}{Q_L} = \\frac{1}{Q_e} + \\frac{1}{Q_0} \n",
+    "$$\n",
+    "\n",
+    "If one defines the _coupling factor_ $\\beta=Q_0/Q_e$ then:\n",
+    "\n",
+    "$$\n",
+    "Q_0 = (1 + \\beta) Q_L\n",
+    "$$\n",
+    "\n",
+    "The half-power _fractional bandwidth_ $BW$ (aka 3dB bandwidth) of the resonator is defined by:\n",
+    "$$\n",
+    "BW=\\frac{\\Delta f}{f_r}\n",
+    "$$\n",
+    "\n",
+    "Then\n",
+    "\n",
+    "$$\n",
+    "BW = \\frac{1}{Q_L}\n",
+    "$$\n",
+    "\n",
+    "While the measurements of the loaded Quality factor is straightforward, obtaining uncertainty below 1% (which is considered to be low for Q-factor measurement) requires attention to several aspects of the experimental procedure.\n",
+    "\n",
+    "`scikit-rf` implements the methods described in [3] for determining loaded and unloaded Q-factors from frequency-domain S-parameters, that can be applied to measurements of transmission or reflection.\n",
+    "\n",
+    "## Example with a Parallel RLC Circuit\n",
+    "To illustrate the usage of the `Qfactor` class of `scikit-rf`, a parallel RLC circuit as illustrated above is used as an canonical example for which analytical formulas are available for benchmarking.\n",
```

Comment:
That section renders incorrectly. The line break at the end is not working (at least in the Github preview).